pypeit/par/util.py

```diff
@@ -10,8 +10,8 @@
 
 from configobj import ConfigObj
 
-from pypeit import msgs, __version__
-from pypeit.pypmsgs import PypeItError
+from pypeit import log, __version__
```

Comment:
Unused.

Reply:
Good call!  I also removed the numpy import (also not used).